experimental/dds/tree2/src/test/shared-tree/editing.spec.ts

```diff
@@ -489,6 +489,288 @@ describe("Editing", () => {
 			expectJsonTree([tree, tree2], ["B", "A", "D", "C"]);
 		});
 
+		it("destination of move's ancestor deleted", () => {
```

Comment:
Maybe add a comment describing the intention of this test. Something like "moving a node into a subtree which has been concurrently deleted results in the moved node also being deleted."